app/models/contribution.rb

```diff
@@ -64,4 +66,10 @@ def update_rejected_reason(rejected_reason = "other_reason", reviewer_id)
   scope :valid_pull_requests, -> { where.not(state: :refused) }
   scope :without_pr_state, ->(state) { where.not(pr_state: state) }
   scope :tracking, -> { where(tracking: true) }
+
+  private
+
+  def normalize_description_blank_values
+    self[:description] = description.blank? ? nil : description
```

Comment:
you can do here `self[:description] = description.presence`

Reply:
on it